configs/calculate_ttbb_lepWeightsUL.py

```diff
@@ -207,20 +224,64 @@ def calculate_variables(event, wrapper, sample, jec = None, dataEra = None, genW
         muIsoSF_up    *= (isosf + isosfErr)
         muIsoSF_down  *= (isosf - isosfErr)
 
+        
         trig    = data[dataEra]["muTrig"].evaluate(
-                    abs(event.Mu_Eta[iMu]), min(199., event.Mu_Pt[iMu]), "nominal")
+                       dataEra+"_UL", abs(event.Mu_Eta_tight[iMu]), max(29.,event.Mu_Pt_tight[iMu]), "sf")
         trigErr = data[dataEra]["muTrig"].evaluate(
-                    abs(event.Mu_Eta[iMu]), min(199., event.Mu_Pt[iMu]), "syst")
-        muTrigSF      *= trig
-        muTrigSF_up   *= (trig+trigErr)
-        muTrigSF_down *= (trig-trigErr)
+                       dataEra+"_UL", abs(event.Mu_Eta_tight[iMu]), max(29.,event.Mu_Pt_tight[iMu]), "syst")
```

Comment:
for muons also a systup and systdown exist which we should use instead of only syst